src/Umbraco.Cms.Integrations.Library/Entities/LibraryMedia.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Text.Json.Serialization;
```

Comment:
Just to validate - do we think this makes sense as a separate package with re-use across various integrations?  Or is it better to fold back into Algolia if it's only used there?  I'm not sure... but just wanted to you to consider before we commit to this.